app/views/dashboards/_table_content.html.erb

```diff
@@ -47,4 +47,7 @@
       <% end %>
     </tbody>
   </table>
+  <% if table_type == 'call_list' && current_user.patients.count > 0%>
+    <%= link_to "Clear your call list", clear_call_list_path(current_user), class:"pull-right text-danger", data: { confirm: "Are you sure you want to clear your current call list?" }, method: :patch, remote: true %>
```

Comment:
I think this will also flatten completed calls. Can you make a note of that in the alert text?